src/schema/top_level_files.yaml

```diff
@@ -0,0 +1,41 @@
+README:
+  required: True
+  extensions:
+    - .md
+    - .rst
```

Comment:
This seems new. Is this encoding current validator practice, or are we proposing that Markdown and RST extensions should be accepted?

Reply:
I can't find a basis for this in the specification, pybids, or the validator, so I'm not sure what I got it from. I will remove these.

Reply:
Removed in 6ce57e1.